src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction_Actions.cs

```diff
@@ -155,38 +147,32 @@ public override Task<Compilation> TransformCompilationAsync(Compilation oldCompi
 
             internal sealed class AddAnalyzerReferencesAction : CompilationAndGeneratorDriverTranslationAction
             {
+#pragma warning disable IDE0052 // Remove unread private members
+                // https://github.com/dotnet/roslyn/issues/44161: right now there is no way to tell a GeneratorDriver that an analyzer reference has been added
```

Comment:
There is `GeneratorDriver.{Add,Remove}Generators()` that can be used for this purpose

Reply:
@chsienki Do we get any benefit from it though at this point?